### ISSUE TITLE
MD-UV380/DM-1701: USB Virtual COM does not work

### ISSUE THREAD

**Issue by Nimayer** [OpenRTX/OpenRTX#4]
USB Virtual COM (serial port over USB) does not work on Tytera MD-UV380 and Baofeng DM-1701

This is the Linux kernel dmesg shown after connecting DM-1701
```
[55737.767635] usb 1-1.4: new full-speed USB device number 30 using xhci_hcd
[55742.855829] xhci_hcd 0000:00:14.0: Timeout while waiting for setup device command
[55748.236025] xhci_hcd 0000:00:14.0: Timeout while waiting for setup device command
[55748.443965] usb 1-1.4: device not accepting address 30, error -62
[55748.444046] usb 1-1-port4: unable to enumerate USB device
```
This is the Linux kernel dmesg after connecting MD-UV380
```
[ 2085.026956] usb 1-12: new full-speed USB device number 4 using xhci_hcd
[ 2085.148375] usb 1-12: device descriptor read/64, error -71
[ 2085.381680] usb 1-12: device descriptor read/64, error -71
[ 2085.614957] usb 1-12: new full-speed USB device number 5 using xhci_hcd
[ 2085.741684] usb 1-12: device descriptor read/64, error -71
[ 2085.978388] usb 1-12: device descriptor read/64, error -71
[ 2086.090196] usb usb1-port12: attempt power cycle
[ 2086.738317] usb 1-12: new full-speed USB device number 6 using xhci_hcd
[ 2086.738436] usb 1-12: Device not responding to setup address.
[ 2086.951060] usb 1-12: Device not responding to setup address.
[ 2087.155017] usb 1-12: device not accepting address 6, error -71
[ 2087.281683] usb 1-12: new full-speed USB device number 7 using xhci_hcd
[ 2087.281828] usb 1-12: Device not responding to setup address.
[ 2087.488452] usb 1-12: Device not responding to setup address.
[ 2087.695278] usb 1-12: device not accepting address 7, error -71
[ 2087.695419] usb usb1-port12: unable to enumerate USB device
```

**Comment by silseva** [comment]
Regression also on MD-380:
`[10955.476495] usb 2-1.2: new full-speed USB device number 108 using ehci-pci
[10955.556538] usb 2-1.2: device descriptor read/64, error -32
[10955.744582] usb 2-1.2: device descriptor read/64, error -32
[10955.932729] usb 2-1.2: new full-speed USB device number 109 using ehci-pci
[10956.012768] usb 2-1.2: device descriptor read/64, error -32
[10956.200871] usb 2-1.2: device descriptor read/64, error -32
[10956.309202] usb 2-1-port2: attempt power cycle
[10956.913224] usb 2-1.2: new full-speed USB device number 110 using ehci-pci
[10957.329437] usb 2-1.2: device not accepting address 110, error -32
[10957.409476] usb 2-1.2: new full-speed USB device number 111 using ehci-pci
[10957.825610] usb 2-1.2: device not accepting address 111, error -32
[10957.825721] usb 2-1-port2: unable to enumerate USB device`

**Comment by silseva** [comment]
Fixed regression for MD-380 in [42e4750](https://github.com/n1zzo/OpenRTX/commit/42e4750245b7b1d25654cc4714b8c330dd62f028).

**Comment by silseva** [comment]
42e4750 also fixed the issue with TYT MD-UV380
